docs/source/kernel-kubernetes.md

```diff
@@ -441,6 +441,19 @@ emptyDir:
 {% endif %}
 ```
 
+### Kubernetes Resource Quotas
+
+When deploy kernel on Kubernetes cluster best practice should define CPU, Memory request and limit. All variable is passthru env when create new kernel.
+
+- `KERNEL_CPUS` - CPU Request by Kernel
+- `KERNEL_MEMORY` - MEMORY Request by Kernel
+- `KERNEL_GPUS` - GPUS Request by Kernel
+- `KERNEL_CPUS_LIMIT` - CPU Limit
+- `KERNEL_MEMORY_LIMIT` - MEMORY Limit
+- `KERNEL_GPUS_LIMIT` - GPUS Limit
+
+Memory and CPU unit-based on [Kubernetes Official Documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) and GPU is using NVIDIA `nvidia.com/gpu` parameter
```

Comment:
```suggestion
Memory and CPU units are based on the [Kubernetes Official Documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/) while GPU is using the NVIDIA `nvidia.com/gpu` parameter.  The desired units should be included in the variable's value.

When defined, these variables are then substituted into the appropriate location of the corresponding kernel-pod.yaml.j2 template.
```

Reply:
I think it might be good to include the section of the jinja template containing these parameters following this paragraph - similar to what is done for the volume mounts.  That way folks see the context of what is being discussed.